clients/kratos/dart/pubspec.yaml

```diff
@@ -1,7 +1,8 @@
 name: ory_kratos_client
 version: 0.11.1
-description: OpenAPI API client
+description: "OpenAPI API client for Ory's next-gen identity server with Ory-hardened authentication, MFA, FIDO2, TOTP, WebAuthn, profile management, identity schemas, social sign in, ..."
```

Comment:
Might then be too long ...

Reply:
It looks good to me @IchordeDionysos 
Is there a maximum length for the description?

Reply:
180 characters :)
https://dart.dev/tools/pub/pubspec#repository:~:text=The%20description%20should%20be%20relatively%20short%E2%80%9460%20to%20180%20characters%E2%80%94and%20tell%20a%20casual%20reader%20what%20they%20might%20want%20to%20know%20about%20your%20package.

Otherwise pub.dev will reduce the score of the package:
![image](https://user-images.githubusercontent.com/10195482/231733355-fe325658-6e1a-48f7-a7df-9687056ba3c3.png)

Leading to it being less likely to be found:
![image](https://user-images.githubusercontent.com/10195482/231733534-af71c5bf-0d72-469b-b34d-fe3c83989717.png)
All of the packages have a higher score than `ory_client`

Reply:
In that case I would suggest this:

```suggestion
description: "OpenAPI API client for Ory Kratos, Ory's central identity management solution with hardened authentication, MFA, FIDO2, TOTP, WebAuthn, SSO, profile management, and more."
```

170 characters and SSO basically includes social login